docs/setup/new_environment.md

```diff
@@ -373,6 +373,9 @@ where `{email}` is the email address you would like to use as the username.
 
 ### Add a new CommCare build
 
-In order to create new versions of applications created in the CommCareHQ app builder, you will need to add the latest `CommCare Mobile` and `CommCare Core` builds to your server.
+In order to create new versions of applications created in the CommCareHQ app builder, you will need to add the the latest `CommCare Mobile` and `CommCare Core` builds to your server. You can do this by running the command below - it wil fetch the latest version from Github.
+
+``` bash
+$ cchq monolith django-manage add_commcare_build --latest
```

Comment:
It looks like we aren't consistent in this doc about using `cchq` or `commcare-cloud`... A bit out of scope for this PR, but would you also want to switch all references of `cchq` to `commcare-cloud`, since I think `cchq` needs to be set up as an alias.